apps/pano/src/models/user.server.ts

```diff
@@ -119,3 +119,23 @@ export const isOwner = <T extends Entity>(
 ): entity is T => {
   return !!user && !!entity && entity.owner.id === user.id;
 };
+
+export async function changeTheme(user: User, theme: UserPreference["theme"]) {
+  // update or create user preference
+  await prisma.userPreference.upsert({
```

Comment:
ideally this should be just an `update` because, we ideally need to create a new `UserPreference` record at `User` creation time.

so, let's update the seed file to create a user preference record after creating the users there.